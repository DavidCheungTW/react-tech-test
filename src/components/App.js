import React from "react";
import "../styles/app.css";

const App = () => {
  return <div className="app">React Tech Test</div>;
};

export default App;
