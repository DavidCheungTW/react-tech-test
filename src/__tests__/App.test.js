import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import App from "../components/App";

describe("App", () => {
  test("Renders as expected", () => {
    const rendered = renderer.create(<App />);

    expect(rendered).toMatchSnapshot();
  });

  test("renders learn react link", () => {
    render(<App />);

    expect(screen.getByText(/React Tech Test/i)).toBeInTheDocument();
  });
});
