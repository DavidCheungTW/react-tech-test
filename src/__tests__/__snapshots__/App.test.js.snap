// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders as expected 1`] = `
<div
  className="app"
>
  React Tech Test
</div>
`;
